docs/modules/kotlin-binding/pages/pkl-config-kotlin.adoc

```diff
@@ -20,40 +20,41 @@ To use the library in a Gradle project, declare the following dependency:
 
 [tabs]
 ====
-Groovy::
+Kotlin::
 +
-.build.gradle
-[source,groovy,subs="+attributes"]
+.build.gradle.kts
+[source,kotlin,subs="+attributes"]
 ----
 dependencies {
-  compile "org.pkl-lang:pkl-config-kotlin:{pkl-artifact-version}"
+  implementation("org.pkl-lang:pkl-config-kotlin:{pkl-artifact-version}")
 }
 
 repositories {
 ifdef::is-release-version[]
   mavenCentral()
 endif::[]
 ifndef::is-release-version[]
-  maven { url "{uri-sonatype}" }
+  maven { url = uri("{uri-sonatype}") }
```

Comment:
What about using this one?

```suggestion
  maven(url = "{uri-sonatype}")
```

Looks IMHO slightly better.

Reply:
definitely